_posts/2020-09-30-sparklyr-1.4.0-released/sparklyr-1.4.0.Rmd

```diff
@@ -0,0 +1,368 @@
+---
+title: "sparklyr 1.4: Weighted Sampling, Tidyr Verbs, Robust Scaler, RAPIDS, and more"
+description: |
+  Sparklyr 1.4 is now available! This release comes with delightful new features such as weighted sampling and tidyr verbs support for Spark dataframes, robust scaler for standardizing data based on median and interquartile range, spark_connect interface for RAPIDS GPU acceleration plugin, as well as a number of dplyr-related improvements.
+author:
+  - name: Yitao Li
+    url: https://github.com/yitao-li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+slug: sparklyr-1.4
+date: 09-30-2020
+categories:
+  - R
+  - Packages/Releases
+  - Distributed Computing
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/sparklyr-1.4.png
```

Comment:
For the thumbnail, since you want the equation (which I think it's great), you could also consider writing it yourself on a piece of paper and taking a photo. I'm just saying this since the equation looks lonely compared to the other post thumbnails. It could also help to make it a bit bigger and keep the image with a bit higher quality; we definitely don't want 2MB thumbnails, but we can probably spare 100KB. Feel free to ignore this since it's really picky feedback!

Reply:
Well, given this will sit directly above `torch` (which isn't so big either), size isn't necessarily much of a problem I'd say ... but I agree that the resolution could be a bit higher ...